run-tests.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cd terminado
+python -m unittest discover $@
```

Comment:
Is there a specific reason to do this instead of running the tests with py.test or nose?